Configuration/Flexforms/Treeview.xml

```diff
@@ -0,0 +1,356 @@
+ *
+ * This file is part of the Kitodo and TYPO3 projects.
+ *
+ * @license GNU General Public License version 3 or later.
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+-->
+<T3DataStructure>
+    <meta>
+        <langDisable>1</langDisable>
+    </meta>
+    <sheets>
+        <sDEF>
+            <ROOT>
+                <TCEforms>
+                    <sheetTitle>LLL:EXT:dlf/Resources/Private/Language/Treeview.xml:tt_content.pi_flexform.sheet_general</sheetTitle>
+                </TCEforms>
+                <type>array</type>
+                <el>
+                    <apikey>
+                        <TCEforms>
+                            <exclude>1</exclude>
+                            <label>LLL:EXT:dlf/Resources/Private/Language/Treeview.xml:tt_content.pi_flexform.apikey</label>
+                            <config>
+                                <type>input</type>
+                                <size>10</size>
+                                <max>30</max>
+                                <eval>trim,required</eval>
+                            </config>
+                        </TCEforms>
+                    </apikey>
+                    <dataFormatter>
+                        <TCEforms>
+                            <onChange>reload</onChange>
+                            <displayCond>FIELD:pages:REQ:true</displayCond>
+                            <exclude>1</exclude>
+                            <label>LLL:EXT:dlf/Resources/Private/Language/Treeview.xml:tt_content.pi_flexform.data_formatter</label>
+                            <config>
+                                <type>select</type>
+                                <items type="array">
+                                    <numIndex index="0" type="array">
+                                        <numIndex index="0">Signatur Hierarchie</numIndex>
+                                        <numIndex index="1">signature</numIndex>
+                                    </numIndex>
+                                    <numIndex index="1" type="array">
+                                        <numIndex index="0">Kollektionen gruppiert nach Jahreszahlen</numIndex>
+                                        <numIndex index="1">collection</numIndex>
+                                    </numIndex>
+                                </items>
+                                <size>1</size>
+                                <maxitems>1</maxitems>
+                                <minitems>0</minitems>
+                            </config>
+                        </TCEforms>
+                    </dataFormatter>
+                    <field>
+                        <TCEforms>
+                            <exclude>1</exclude>
+                            <label>LLL:EXT:dlf/Resources/Private/Language/Treeview.xml:tt_content.pi_flexform.field</label>
+                            <config>
```

Comment:
Since this only works with fields available in Solr and configured in a certain way, instead of having an input field this should be a selector which provides all applicable fields to select from.